table/meta_blocks.cc

```diff
@@ -172,6 +173,8 @@ Status ReadProperties(const Slice &handle_value, RandomAccessFile *file,
       {TablePropertiesNames::kFixedKeyLen,
        &new_table_properties->fixed_key_len}, };
 
+  new_table_properties->compression_type = found_compression_type;
```

Comment:
maybe we can also say the percentage of blocks that are compressed vs. blocks that aren't (because of that logic): https://github.com/facebook/rocksdb/blob/master/table/block_based_table_builder.cc#L332